modules/Departments/department_course_class.php

```diff
@@ -47,23 +49,12 @@
         $page->addError(__('You have not specified one or more required parameters.'));
     } else {
         if (!empty($gibbonDepartmentID)) {
-            $data = array('gibbonCourseClassID' => $gibbonCourseClassID);
-            $sql = "SELECT gibbonCourse.gibbonSchoolYearID,gibbonDepartment.name AS department, gibbonCourse.name AS courseLong, gibbonCourse.nameShort AS course, gibbonCourseClass.name AS classLong, gibbonCourseClass.nameShort AS class, gibbonCourse.gibbonCourseID, gibbonSchoolYear.name AS year, gibbonCourseClass.attendance, gibbonCourseClass.fields
-                    FROM gibbonCourse
-                    JOIN gibbonCourseClass ON (gibbonCourse.gibbonCourseID=gibbonCourseClass.gibbonCourseID)
-                    JOIN gibbonSchoolYear ON (gibbonCourse.gibbonSchoolYearID=gibbonSchoolYear.gibbonSchoolYearID)
-                    JOIN gibbonDepartment ON (gibbonDepartment.gibbonDepartmentID=gibbonCourse.gibbonDepartmentID)
-                    WHERE gibbonCourseClassID=:gibbonCourseClassID";
+            $result = $courseGateway->getCourseClassInfoAndDepartment($gibbonCourseClassID);
```

Comment:
We've used the term Details in other methods, so this would help with consistency.
```suggestion
            $result = $courseGateway->getCourseClassDetails($gibbonCourseClassID);
```